Mathlib/Topology/ShrinkingLemma.lean

```diff
@@ -255,3 +255,122 @@ theorem exists_iUnion_eq_closed_subset (uo : ∀ i, IsOpen (u i)) (uf : ∀ x, {
   ⟨v, univ_subset_iff.1 vU, hv⟩
 
 end NormalSpace
+
+section T2LocallyCompactSpace
+
+open ShrinkingLemma
+
+variable {u : ι → Set X} {s : Set X} [T2Space X] [LocallyCompactSpace X]
+
+/-- In a locally compact Hausdorff space `X`, if `s` is a compact set, `v` is a partial refinement,
+and `i` is an index such that `i ∉ v.carrier`, then there exists a partial refinement that is
+strictly greater than `v`. -/
+theorem exists_gt_t2space (v : PartialRefinement u s (fun w => IsCompact (closure w)))
+    (hs : IsCompact s) (i : ι) (hi : i ∉ v.carrier) :
```

Comment:
```suggestion
    (hs : IsCompact s) (i : ι) (hi : i ∉ v) :
```

Reply:
this gives an error "failed to synthesize Membership ι (PartialRefinement u s fun w ↦ IsCompact (closure w))", I revert it.